VSTS/SonarQube/Tasks/Common/common/parameter-helper.ts

```diff
@@ -0,0 +1,103 @@
+
+import tl = require("vsts-task-lib/task");
+
+import {ToolRunner} from "vsts-task-lib/toolrunner";
+import {SonarQubeEndpoint} from "./endpoint";
+import {Utils} from "./utils";
+import {VstsServerUtils} from "./vsts-utils";
+
+export class SonarQubeParameterHelper {
+
+     /**
+      * Applies parameters for SonarQube features enabled by the user.
+      * @param toolRunner     ToolRunner to add parameters to
+      * @returns {ToolRunner} ToolRunner with parameters added
+      */
+    public static addSonarQubeParameters(toolRunner: ToolRunner): ToolRunner {
```

Comment:
This is just an idea, don't take it as if you should really change the code: I would generally use shorter method names, especially when most parts of the name are the same as the class name:
addSonarQubeParameters(toolRunner) --> setup(toolRunner) // or initialize() or something better...
addSonarQubeSourcesParams(toolRunner) --> addSources(toolRunner) // or may be setSources()?
...